index.html

```diff
@@ -94,17 +143,121 @@ <h3>Professional Art Printing Data</h3>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard.
+        </p>
+      
+        <ul class="languages">
+         
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>ruby</li>
+        </ul>
+      </div>
+
+        <button  type="submit">See Project</button>
+      </div>
+    
+
+      <div class="bg-page-two">
+        <h3>Professional Art Printing Data</h3>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="languages">
+ 
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>ruby</li>
+        </ul>
+
+        <button class="without-button" type="submit">See Project</button>
+      </div>
+
+      <div class="bg-page-three">
+        <h3>Professional Art Printing Data</h3>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="languages">
+         
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>ruby</li>
+        </ul>
+
+        <button class="without-button" type="submit">See Project</button>
+      </div>
+      <div class="bg-page-two">
+        <h3>Professional Art Printing Data</h3>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="languages">
+         
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>ruby</li>
+        </ul>
+
+        <button class="without-button" type="submit">See Project</button>
+      </div>
+
+      <div class="bg-page-three">
+        <h3>Professional Art Printing Data</h3>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="languages">
+          
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>ruby</li>
+        </ul>
+
+        <button class="without-button" type="submit">See Project</button>
+      </div>
+      <div class="bg-page-three">
+        <h3>Professional Art Printing Data</h3>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="languages">
+         
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>ruby</li>
+        </ul>
+
+        <button class="without-button" type="submit">See Project</button>
       </div>
+      
+    </div>
+      
 
       <section class="about">
-        <div class="about-me">
-          <div></div>
+        <div class="about-me-header">
+         
           <h1>About me</h1>
           <p>
```

Comment:
- [ ] Please there are 4 to 5 lines of text describing the student's experience. I suggest that you assign a "width: 100%" to `<p>` to adjust the text a little bit